### ISSUE TITLE
Improve screen reader support

### ISSUE THREAD

**Issue by wavebeem** [wavebeem/pkmn.help#113]
- Pokedex "help" link should have an aria-label
- Offense "edit" link should have an aria-label
- Offense type chooser
  - Use an aria-hidden to avoid announcing the checkmark text
  - Change them to actual checkbox elements?
- Defense type chooser
  - Use aria-hidden to avoid announcing the selected circle text
  - Use radio button roles (or actual radio buttons?)
  - Make keyboard nav work like radio buttons
  - Probably best to just make them label+input[type=radio] elements, and use appearance:none to completely override the styles :D
- Pokedex
  - Offense/defense links should omit form name if not defined
